.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "tools/OpenROAD"]
 	path = tools/OpenROAD
 	url = ../OpenROAD.git
+[submodule "tools/LSOracle-Plugin"]
+	path = tools/LSOracle-Plugin
+	url = git@github.com:lnis-uofu/LSOracle-Plugin.git
```

Comment:
can you change to the https url instead?

Reply:
why is https preferred?

Reply:
Jenkins does not like to use ssh when it does not have r/w access to the repo. That is the reason the pipeline checks are failing now.